samples/drivers/led_lp3943/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _lp3943:
+
+LP3943: 16-Channel RGB, White-LED Driver
+########################################
+
+Overview
+********
+
+This sample controls 16 LEDs connected to an LP3943 driver, in
+a continuous pattern of turning them on one at a time (at a one
+second interval) until they're all on, and then turning them off in
+reverse order.
+
+Requirements
+************
+
+The :ref:`96b_neonkey`  board has an LP3943 driver on board, so we'll use
```

Comment:
Ah.  I missed that this board also has the LEDs being controlled as well (so you don't need to connect an LED strip for example, as I asked).  We should mention that:

    The :ref:`96b_neonkey`  board has an LP3943 driver and 16 LEDs on board, ...

Reply:
okay